src/Utils/fetch.js

```diff
@@ -3,8 +3,10 @@ export const fetchData = async (path, options) => {
     if(response.status >= 300) {
       const error = await response.json()
       throw new Error(error)
-    } else {
+    } else if (response.status !== 204) {
       return await response.json()
+    } else {
+      return response.status;
     }
```

Comment:
I like that you are now returning the status if it is the only thing that has been returned.  I'm not sure if we will be using it but we will be happy when we do need it! 😀🔥